ctgan/data_transformer.py

```diff
@@ -139,39 +128,23 @@ def transform(self, raw_data):
 
         column_data_list = []
         for column_transform_info in self._column_transform_info_list:
-            column_data = raw_data[[column_transform_info.column_name]].to_numpy()
+            column_name = column_transform_info.column_name
+            data = raw_data[[column_name]]
             if column_transform_info.column_type == 'continuous':
-                column_data_list += self._transform_continuous(column_transform_info, column_data)
+                column_data_list.append(self._transform_continuous(column_transform_info, data))
             else:
-                assert column_transform_info.column_type == 'discrete'
-                column_data_list += self._transform_discrete(column_transform_info, column_data)
+                column_data_list.append(self._transform_discrete(column_transform_info, data))
 
         return np.concatenate(column_data_list, axis=1).astype(float)
 
     def _inverse_transform_continuous(self, column_transform_info, column_data, sigmas, st):
         gm = column_transform_info.transform
-        valid_component_indicator = column_transform_info.transform_aux
-
-        selected_normalized_value = column_data[:, 0]
-        selected_component_probs = column_data[:, 1:]
-
+        data = pd.DataFrame(column_data[:, :2], columns=list(gm.get_output_types()))
+        data.iloc[:, 1] = np.argmax(column_data[:, 1:], axis=1)
         if sigmas is not None:
-            sig = sigmas[st]
-            selected_normalized_value = np.random.normal(selected_normalized_value, sig)
+            data.iloc[:, 0] = np.random.normal(data.iloc[:, 0], sigmas[st])
```

Comment:
minor: for readability can we pull `data.iloc[:, 0]` into a variable with a descriptive name?